docker/serverd.sh

```diff
@@ -0,0 +1,2 @@
+while [ ! -f /var/rudder/cfengine-community/inputs/promises.cf ]; do sleep 1; done && /opt/rudder/bin/cf-serverd -FI
```

Comment:
could you use long options `--no-fork` and `--inform`? Short options are good for use in the CLI but using long options in scripts makes them more readable.